virttest/storage.py

```diff
@@ -321,22 +322,31 @@ def __init__(self, image, data, data_format, storage_type, **info):
         :param image: image tag name
         :param data: sensitive data like password
         :param data_format: raw or base64
-        :param storage_type: ceph, gluster or iscsi-direct
+        :param storage_type: ceph, glusterfs-direct or iscsi-direct
         :param info: other access information, such as:
                      iscsi-direct: initiator
-                     gluster: debug, logfile
+                     gluster-direct: debug, logfile, peers, port, transport
         """
         self.image = image
         self.aid = '%s_access_secret' % self.image
         self.storage_type = storage_type
         self.filename = os.path.join(secret_dir, "%s.secret" % self.aid)
+        self.data_format = data_format
 
         if self.storage_type == 'iscsi-direct':
             self._chap_passwd = data
-            self.iscsi_initiator = info.get('initiator')
+            self.iscsi_initiator = info['initiator']
```

Comment:
Looks not be relevant with the gluster stuff.

Reply:
yes,  have changed it during code review, remove the change from the gluster patch